fmt.sh

```diff
@@ -72,11 +79,13 @@ USAGE
       javafmt_args=(--set-exit-if-changed --dry-run)
       dade_copyright_arg=check
       buildifier_target=//:buildifier
+      buf_args=(check breaking --against "${buf_image}")
       ;;
     --diff)
       shift
       merge_base="$(git merge-base origin/main HEAD)"
       scalafmt_args+=('--mode=diff' "--diff-branch=${merge_base}")
+      buf_args=(check breaking --against "${buf_image}")
```

Comment:
I’m not sure this makes sense. `--diff` shouldn’t change the semantics. It’s a fast version of running without `--diff` that only checks files that changed. With your PR `--diff` acts like `--test` for `buf`. Given that people tend to want to run `./fmt.sh` before committing, I don’t think this should update the image. Instead, I’d put that in a separate script or a separate flag in this script and then default to checking the diff for both no flag, `--diff` and `--test`.

Reply:
Good point. Added an option `rebuild-buf-image` to `fmt.sh` and replaced the default behaviour to be always just running a `buf check breaking`.